ReactAndroid/build.gradle

```diff
@@ -240,6 +240,7 @@ dependencies {
     compile 'com.squareup.okhttp:okhttp-ws:2.4.0'
     compile 'com.squareup.okio:okio:1.5.0'
     compile 'org.webkit:android-jsc:r174650'
+    compile "org.java-websocket:Java-WebSocket:1.3.0"
```

Comment:
@foghina From the README on the [okhttp-ws page](https://github.com/square/okhttp/tree/master/okhttp-ws)

> Note: This module's API should be considered experimental and may be subject to breaking changes in future releases.

Currently it's used to connect to Chrome debugger I think. The docs don't have anything and the API is also subject to change. Should I migrate it to use `okhttp-ws` ? I'm a bit skeptical.